@rakannimer let me know if there's anything else that I should do as part of this PR. I tried to run the tests locally but I got the following error:

```
docz-core:   ● Test suite failed to run
docz-core:     Cannot find module 'docz-core' from '/Users/pedropinera/src/github.com/doczjs/docz/core/docz-core/src/config'
docz-core:     However, Jest was able to find:
docz-core:      '../config/paths.ts'
```

--- a/core/gatsby-theme-docz/src/wrapper.js
+++ b/core/gatsby-theme-docz/src/wrapper.js
@@ -1,6 +1,6 @@
 import * as React from 'react'
 
-const Wrapper = ({ children, doc }) => (
+const Wrapper = ({ children }) => (
   <React.Fragment>{children}</React.Fragment>
 )
 export default Wrapper

